src/check_user_num.js

```diff
@@ -0,0 +1,45 @@
+export { createUserNumbers };
+
+const userInput = document.querySelector('#user-input');
+
+// 만약 사용자 입력값에 오류가 있다면 경고창을 띄우고,
+// 오류가 없다면 사용자 입력값 배열로 받아오기
+function createUserNumbers() {
+  const userEnterNumbers = userInput.value;
+  if (userEnterNumbers === '') {
+    alert('숫자를 입력해주세요.');
+    userInput.focus();
+    return;
+  } else if (
+    !isOnlyThreeDigitNumber(userEnterNumbers) ||
+    !isNonOverlap(userEnterNumbers)
+  ) {
+    alert('1에서 9 사이의 중복되지 않는 숫자 3개를 입력해주세요.');
+    userInput.focus();
+    return;
+  } else {
+    const str = new String(userEnterNumbers);
+    const userNumbers = [...str];
+    return userNumbers;
```

Comment:
이런 부분도 축약해서 사용해도 될거 같네요 ㅎㅎ
```suggestion
    return [...String(userEnterNumbers)];
```